index.html

```diff
@@ -202,13 +194,9 @@ <h2>Terminology</h2>
         <dt><dfn>OCFL Object</dfn>:</dt>
         <dd></dd>
         <dt><dfn>OCFL Object Root</dfn>:<dt>
-        <dd></dd>
+        <dd>The root of an <a>OCFL Object</a>.</dd>
```

Comment:
`The base directory of an OCFL Object that may exist either as a first class directory, or in the case of a zipped OCFL Object, as the top directory in a zip file`.